src/components/transactions/TxDetails/TxData/Rejection/index.tsx

```diff
@@ -19,10 +19,9 @@ const RejectionTxInfo = ({ nonce, isTxExecuted }: Props) => {
 
   return (
     <>
-      <Typography>{message}</Typography>
+      <Typography mr={2}>{message}</Typography>
       {!isTxExecuted && (
-        <>
-          <br />
+        <Box mt={2} sx={{ width: 'fit-content' }}>
```

Comment:
What does the `fit-content` do? In Chrome it looks the same with and without.

Reply:
To not have the link occupying the whole width

Reply:
Addresses all this clickable area

<img width="658" alt="Screenshot 2022-09-29 at 11 17 31" src="https://user-images.githubusercontent.com/32431609/193003086-825e3009-cb90-48df-ada0-cd0ec84cc384.png">

Reply:
Addresses all this clickable area

<img width="791" alt="Screenshot 2022-09-29 at 12 04 53" src="https://user-images.githubusercontent.com/32431609/193003439-0887753a-8506-4e10-ae19-3c142e6aaf44.png">

Reply:
Ah that makes sense, thanks!